Running-Mastodon/Production-guide.md

```diff
@@ -335,6 +335,59 @@ WantedBy=multi-user.target
 
 This allows you to `sudo systemctl enable /etc/systemd/system/mastodon-*.service` and `sudo systemctl start mastodon-web.service mastodon-sidekiq.service mastodon-streaming.service` to get things going.
 
+## Let's Encrypt
+
+This section is only relevant if you are using [Let's Encrypt](https://letsencrypt.org/)
+as your TLS certificate provider.
+
+Other assumptions - Ubuntu 16.04, letsencrypt tool installed from distro repositories.
+
+### Installation of tool
+
+This is how you install the `letsencrypt` package:
+
+`sudo apt -y install letsencrypt`
+
+### Generation of certificate
+
+This is the command you should use to generate a Let's Encrypt certificate.
+Make sure to replace any instances of 'example.com' with your Mastodon instance's domain.
+
+Additional note: This command will require that nginx or another web server is correctly
+configured with your Mastodon instance's domain.
+
+`letsencrypt certonly --webroot -d example.com -w /home/mastodon/live/public/`
+
+### Automated renewal of Let's Encrypt certificate
+
+Let's Encrypt certificates have a validity period of 90 days.
+
+You need to renew your certificate before the expiration date. Failure to do so will
+result in your users being unable to access your instance and other instances being unable 
+to federate with yours.
+
+We can do this with a cron job that runs daily:
+
+`nano /etc/cron.daily/letsencrypt-renew`
+
+Copy and paste this script into that file:
+
+```
+#!/usr/bin/env bash
+letsencrypt renew
+service nginx reload
```

Comment:
`systemctl reload nginx`? Both ways of writing it work but Ubuntu 16.04 is systemd-land and we use systemctl in other parts of the documentation. Same comment for the cron restart lower down.